### ISSUE TITLE
Browser Support and Functional Roadmap?

### ISSUE THREAD

**Issue by workmanw** [alohaeditor/Aloha-Editor#1323]
I apologize if this isn't the right place to ask. I imagine it isn't, but I didn't know where else to ask.

I'm auditing wide range of WYSIWYG editors in hopes of finding a replacement for our current one (redactor). I've spent the past few years maintaining our internal usage of redactor, submitting bugs, etc. And after looking through other editors it was starting to feel like they're all "6 in one, half dozen in another". Really, it's just who claims to manage and normalize contentEditable the best. Until I came across Aloha. I would love nothing more than to use an editor without contentEditable.

I've spent the last few hours looking through Aloha's guides, reading blog posts, and hacking around with demos. It's pretty clear to me things are still Alpha which is perfectly understandable. I'm giving serious thought to being an early adopter of Aloha. But I two fundamental questions. 

1) What browsers are being targeted for Aloha 2
2) Is there a roadmap or any sort of detailed status of what's been done so far and what's left to do. Is there any target time frame for Aloha 2 being released.

Thanks a lot!

**Comment by butsjoh** [comment]
@workmanw Did you get an answer to you question of which browsers are targeted?

**Comment by workmanw** [comment]
@butsjoh I didn't ... sorry.

**Comment by butsjoh** [comment]
Hmm ok says enough then :) Did you in the end found a good alternative? I am kinda doing the same task as you where describing. We are seriously thinking to avoid content editable completly and move towards http://madebymany.github.io/sir-trevor-js/.

**Comment by workmanw** [comment]
@butsjoh TL;DR; Redactor was the best choice for our project (http://imperavi.com/redactor/). I really really wanted to use Aloha. I tried everything to make it work for us, but it's just too alpha. I couldn't justify using it at this stage. 

========

Here is a review of my thought process I wrote up for my team:

I looked at every WYSIWYG editor on the surface. Then narrowed it down to the following 8 WYSIWYG editors: Scribe (written by the Guarian), Quaill.js, SCEditor, CKEditor, Aloha Editor, Froala, Raptor, and Redactor.

Then I narrowed it down again to 4 WYSIWYG editors: Aloha Editor, Froala, Raptor, and Redactor.

Then I looked deeply across the feature set and customization options of each editor. I found them all to be lacking everything we needed for our project. There simply wasn't a clear winner. There is going to be a moderate amount of custom work to needed regardless of which editor we picked.

So given that I did another pass at looking through the API and the flexibility offered by each of the 4 editors. When it came down to the API, redactor was the clear winner. Despite the fact that they've simplified their "on the surface" offering. Their API was completely rewritten, yet again, to have a minimal core and a module system. Allowing us to implement our own modules to define our own interactions for some components.

======

PS Sorry to plug another WYSIWYG editor in this issue. Just trying to help @butsjoh out. I genuinely hope Aloha 2 succeeds and we can ride our selves of contenteditable.
